### PR TITLE
Add basic completion for Keybase

### PR DESCRIPTION
**Pull request by 239**
## Description

Added first stage Keybase commands.

Fixes issue #

## TODOs:

- [ ] Changes to fish usage are reflected in user documentation/manpages.
- [ ] Tests have been added for regressions fixed
- [x] User-visible changes noted in CHANGELOG.md

### CODE REVIEW

**Comment by ridiculousfish** [comment]
@239  this doesn't merge cleanly after the merge of #6245, can you please rebase (and squash while you're at it?) Thanks!

File: share/completions/keybase.fish [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+#variables
+set -l seen __fish_seen_subcommand_from
+set -l commands account blocks bot chat ctl currency decrypt deprovision device encrypt follow fs git help id list-followers list-following log login logout oneshot paperkey passphrase pgp ping prove rekey selfprovision service sign signup sigs status team track unfollow untrack update verify version wallet
+
+#global options
+#...
+
+#commands
+complete -c keybase -f -n "not $seen $commands" -a "$commands"
+#complete -c keybase -f -n "not $seen $commands" -a account -d "Modify your account"
```
Comment (faho) [inline share/completions/keybase.fish]: Any reason why these are all commented out?

Comment (239) [reply share/completions/keybase.fish]: I had started adding option descriptions but there are too many so I wanted just to add the completions for now.

**Comment by 239** [comment]
I have reworked the completions completely so this is not relevant anymore.

**Pull request closed without merging**